app/src/main/java/app/tivi/inject/AppModule.kt

```diff
@@ -24,34 +25,38 @@ import androidx.lifecycle.coroutineScope
 import androidx.navigation.ui.AppBarConfiguration
 import androidx.preference.PreferenceManager
 import app.tivi.BuildConfig
-import app.tivi.TiviApplication
-import app.tivi.extensions.toThreeTenDateTimeFormatter
+import app.tivi.extensions.withLocale
 import app.tivi.home.followed.R
+import app.tivi.tmdb.TmdbModule
+import app.tivi.trakt.TraktModule
 import app.tivi.util.AppCoroutineDispatchers
 import com.google.firebase.crashlytics.FirebaseCrashlytics
 import dagger.Module
 import dagger.Provides
-import io.reactivex.disposables.CompositeDisposable
+import dagger.hilt.InstallIn
+import dagger.hilt.android.components.ApplicationComponent
+import dagger.hilt.android.qualifiers.ApplicationContext
 import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.Dispatchers
-import org.threeten.bp.ZoneId
 import org.threeten.bp.format.DateTimeFormatter
+import org.threeten.bp.format.FormatStyle
 import java.io.File
-import java.text.SimpleDateFormat
 import java.util.concurrent.Executor
 import java.util.concurrent.Executors
 import javax.inject.Named
 import javax.inject.Singleton
-import android.text.format.DateFormat as AndroidDateFormat
 
-@Module(includes = [AppModuleBinds::class])
+@InstallIn(ApplicationComponent::class)
+@Module(
+    includes = [
+        TraktModule::class,
+        TmdbModule::class
+    ]
+)
 class AppModule {
```

Comment:
Better performance if modules with `@Provides` are declared as `object` instead of `class`.

Ditto for the rest of the project

Reply:
Thanks for the reminder.